docs/articles/mirroring.md

```diff
@@ -0,0 +1,432 @@
+- This registry synchronizes with upstream registry every 6 hours. 
+- On-demand mirroring is disabled.
+- Based on the content filtering options, this registry synchronizes these images:
+    - From /repo1/repo, images with tags that begin with "4." and are semver compliant. <br/>Files are stored locally in /repo1/repo on localhost.
+    - From /repo2/repo, images with all tags. <br/>Because `stripPrefix` is enabled, files are stored locally in /repo2. For example, docker://upstream/repo2/repo:v1 is stored as docker://local/repo2:v1.
+    - From /repo3/repo, images with all tags. <br/>Files are stored locally in /repo3/repo.
+
+## Example: Multiple registries with on-demand mirroring
+
+The following is an example of sync configuration for mirroring multiple registries with on-demand mirroring.
+
+```
+{
+  "distSpecVersion": "1.0.1",
+  "storage": {
+    "rootDirectory": "/tmp/zot",
+    "gc": true
+  },
+  "http": {
+    "address": "0.0.0.0",
+    "port": "8080"
+  },
+  "log": {
+    "level": "debug"
+  },
+  "extensions": {
+    "sync": {
+      "enable": true,
+      "registries": [
+        {
+          "urls": ["https://k8s.gcr.io"],
+          "content": [
+            {
+              "prefix": "**", 
+              "destination": "/k8s-images"
+            }
+          ],
+          "onDemand": true,
+          "tlsVerify": true
+        },
+        {
+          "urls": ["https://docker.io/library"],
+          "content": [
+            {
+              "prefix": "**", 
+              "destination": "/docker-images"
+            }
+          ],
+          "onDemand": true,
+          "tlsVerify": true
+        }
+      ]
+    }
+  }
+}
+```
+
+With this zot configuration, the sync behavior is as follows:
+
+1. This user request for content from the zot registry:<br/>
+   `skopeo copy --src-tls-verify=false docker://localhost:8080/docker-images/alpine <dest>`<br/>causes zot to synchronize the content with this remote registry:<br/>&nbsp;&nbsp;&nbsp;&nbsp;`docker.io/library/alpine:latest`<br/>to the zot registry:<br>&nbsp;&nbsp;&nbsp;&nbsp;`localhost:8080/docker-images/alpine:latest`<br/>before delivering the content to the requestor at `<dest>`.
+
+2. This user request for content from the zot registry:<br/>
+  `skopeo copy --src-tls-verify=false docker://localhost:8080/k8s-images/kube-proxy:v1.19.2 <dest>`<br/>causes zot to synchronize the content with this remote registry:<br/>&nbsp;&nbsp;&nbsp;&nbsp;`k8s.gcr.io/kube-proxy:v1.19.2` <br/>to the zot registry:<br/>&nbsp;&nbsp;&nbsp;&nbsp;`localhost:8080/k8s-images/kube-proxy:v1.19.2`<br/>before delivering the content to the requestor at `<dest>`.
+
+You can use this command:<br/>&nbsp;&nbsp;&nbsp;&nbsp;
+`curl http://localhost:8080/v2/_catalog`<br/>to display the local repositories:
+```
+  {
+    "repositories":[
+      "docker-images/alpine",
+      "k8s-images/kube-proxy"
+    ]
+  }
+``` 
+
+## Example: Multiple registries with mixed mirroring modes
+
+<details>
```

Comment:
Only this example has a dropdown?

Reply:
Sorry, I thought I removed them all.  What are people's opinions on this show/hide function for long config structures?

Reply:
Removed that dropdown, too.  Do people prefer the show/hide function for long config blocks?